qcodes/dataset/sqlite/queries.py

```diff
@@ -829,10 +831,47 @@ def get_guid_from_run_id(conn: ConnectionPlus, run_id: int) -> str:
     Args:
         conn: database connection
         run_id: id of the run
+
+    Returns:
+        The guid of the run_id.
     """
     return select_one_where(conn, "runs", "guid", "run_id", run_id)
 
 
+def get_guid_from_multiple_run_ids(db_path: Union[str, Path],
+                                   run_ids: Iterable[int]) \
+                                   -> List[str]:
+    """
+    Expanding the functionality of get_guid_from_run_id to retrive guids
+    from database and for multiple run ids.
+
+    Args:
+        db_path: The path to the database file.
+        run_ids: An integer iterable of run ids to get their guids.
+
+    Returns:
+        A list of guids belonging to the inserted run_ids.
+    """
+
+    guids: List[str] = []
+
+    try:
+        conn = connect(db_path)
+        for run_id in run_ids:
+            if run_exists(conn=conn, run_id=run_id):
+                run_id_guid = get_guid_from_run_id(conn=conn,
+                                                   run_id=run_id)
```

Comment:
Perhaps we should use captured run_id ? and get_guid_from_run_spec` ?

Reply:
@astafan8 what do you think?

Reply:
@FarBo please correct me if the following is wrong, but it seems to me that the users of this function have the following problem: they were working with run_ids (not captured_run_ids), and then they chose a set of run_ids because of what they want to do next, and they just need the GUIDs of those runs, hence using run_id here is actually what the users want. If the runs the users work on happen to be from a number of different databases, i don't see them working with the captured_run_id as they may not uniquely identify a run, hence my feeling is that they work with run_id. So, i think that using run_ids here is fine.

However, i do think that we should make it explicit in the docstring - that it's `run_id` and not `captured_run_id`.